great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -54,18 +127,50 @@ class PandasDatasourceError(Exception):
     # "read_xml",
 )
 
-_ASSET_MODELS = _generate_pandas_data_asset_models(
+_FILESYSTEM_ASSET_MODELS = _generate_pandas_data_asset_models(
     _FilesystemDataAsset,
-    blacklist=_BLACK_LIST,
+    blacklist=_FILESYSTEM_BLACK_LIST,
+    use_docstring_from_method=True,
+    skip_first_param=True,
+)
+
+_PANDAS_BLACK_LIST = (
+    # "read_csv",
+    # "read_json",
+    # "read_excel",
+    # "read_parquet",
+    # "read_clipboard",
+    # "read_feather",
+    "read_fwf",  # unhandled type
```

Comment:
Tried and failed to figure out the issue here, but we are dealing with it in `PandasFilesystemDatasource` as well.